esp/src/src-react/components/DiskUsage.tsx

```diff
@@ -31,17 +41,78 @@ interface DetailsProps {
 export const Details: React.FunctionComponent<DetailsProps> = ({
     cluster
 }) => {
-    const summary = React.useMemo(() => {
-        const retVal = new DetailsWidget(cluster)
-            .refresh()
-            .on("componentClick", component => {
-                pushUrl(`/machines/${component}/usage`);
+
+    const [data, setData] = React.useState<any[]>([]);
```

Comment:
__hpcc_id does not exist in the `data` (see below) - to help prevent this type of issue we could change this pattern to:
```ts
    type Columns = typeof columns;
    type Row = { __hpcc_id: string } & { [K in keyof Columns]: string | number };
    const [data, setData] = React.useState<Row[]>([]);
```